docs/extend/extend-apiml/api-mediation-oidc-authentication.md

```diff
@@ -0,0 +1,271 @@
+ - For CA ACF2 use [IDMAP User Profile Data Records](https://techdocs.broadcom.com/us/en/ca-mainframe-software/security/ca-acf2-for-z-os/16-0/administrating/administer-records/user-profile-records/idmap-user-profile-records.html).
+
+2. External Mapper user's permissions.
+  The External Mapper functionality is executed on behalf of the calling `externalMapperUser`. By default, this will be the ZOWE runtime user `ZWESVUSR`.   
+  For the identity mapping calls to succeed, the externalMapperUser needs special privileges which may vary per installed ESM type.
+
+  Perform following validations and configuration steps to ensure that externalMapperUser is properly configured:   
+1. Make sure that the user has a default user group set. The default group could be any existing group.   
+   
+  - For Top Secret, execute the following commands to set default group in the externalMapperUser profile:
+   
+    ```
+    TSS ADD(user_id) GROUP(group_id)
+    TSS ADD(user_id) DFLTGRP(group_id)
+    ```
+   
+    **TIP:** If the externalMapperUser is also used to call z/OSMF, make sure that the user is in IZUUSER group. 
+
+
+  - For RACF, validate that the user already has DFLTGRP, which should be set by default     
+
+
+  - For ACF2, execute the following command:
+
+    `#TODO- Find what command is needed`
+
+2. Make sure that the externalMappingUser has READ access to the Master Facility class set in the user profile. 
+  - For Top Secret, execute the following command: 
+
+    ```TSS PERMIT(user) IBMFAC(IRR.IDIDMAP.QUERY) ACCESS(READ)```
+
+
+  - For RACF, it is not required that the externalMapperUser has permissions to the Master Facility class.
+
+    
+  - For ACF2, execute the following command:
+
+    `#TODO- Find what command is needed`
+
+3. Make sure that the externalMapperUser has access to the ZOWE facility.
+  - For Top Secret, execute the following command:
+
+    ```TSS ADDTO(user) FACILITY(ZOWE) ```
+
+
+  - For RACF it is not required that the externalMapperUser has permissions to the ZOWE Facility class.
+   
+    `#TODO- Find what command is needed`
+
+
+  - For ACF2 take the following command:
+
+    `#TODO- Find what command is needed`
+
+4. Make sure that the externalMapperUser has permissions for the OMVSAPPL.
+  - For Top Secret, execute the following command:
+
+    ```TSS PERMIT(user) APPL(OMVSAPPL)```
+   
+
+  - For RACF, execute the following commands:
+    ```
+    PERMIT OMVSAPPL CLASS(APPL) ID(user) ACCESS(READ)
+    SETR RACLIST(APPL) REFRESH
+    ```
+
+  - For ACF2, execute the following command:
+
+    `#TODO- Find what command is needed`
+
+5. Make sure the externalMapperUser has access to the `IRR.IDIDMAP.QUERY` facility.
```

Comment:
This is basically the same as point 2 above.

Reply:
Good catch. Thanks!